lib/components/app/app-menu.tsx

```diff
@@ -200,7 +259,23 @@ class AppMenu extends Component<
           onRequestClose={this._togglePane}
           width="320px"
         >
-          <div className="app-menu">
+          <nav
+            aria-label={intl.formatMessage({
+              id: 'components.AppMenu.appMenu'
+            })}
+            className="app-menu"
+          >
+            {/* This item is duplicated by the view-switcher, but only shown on mobile
+            when the view switcher isn't shown (using css) */}
+            <MenuItem
```

Comment:
`<MenuItem>` resolves to `<li>`, so surround the list items with `<ul>`.

Reply:
Surprised we didn't ready have this!